### ISSUE TITLE
Phoneboard Mac

### ISSUE THREAD

**Issue by xavierartot** [Phoneboard/phoneboard#98]
Hi there,
I'm using Mac OS 11. Do we have the diode value ?
I didn't see any option to display them?
Thanks

**Comment by kegesch** [comment]
Yes, there is an option for diode values under view. But I think there are no boards with values yet, at least I could not find any.
